### ISSUE TITLE
Can't find failed diff test screenshots

### ISSUE THREAD

**Issue by nurovi** [garris/BackstopJS#517]
Hi,

I followed the instructions on the css-tricks site: https://css-tricks.com/automating-css-regression-testing/

With the only modification being to the backstop.json file adjusting the path with `../../` since we are two folders from the root.  The error is occurring when I run the test.  The reference correctly creates screenshots in the backstop_data folder at the root, and the test also creates screenshots under `backstop_data/bitmaps_test/somenumbers`.  The output I get from backstop is this: 
```
> npm run test

> backstopjs@3.0.20 test C:\Source\multi-tenant\mdp.node.seleniumtests\node_modules\backstopjs
> node ./cli/index.js test

BackstopJS v3.0.20
BackstopJS is running in legacy mode.
Loading config:  C:\Source\multi-tenant\mdp.node.seleniumtests\backstop.json

COMMAND | Executing core for `test`
createBitmaps | Selcted 1 of 1 scenarios.
Starting Chromy: port:9222 --disable-gpu,--force-device-scale-factor=1,--window-size=320,480
onBefore.js has run for phone.
onReady.js has run for:  phone
Starting Chromy: port:9223 --disable-gpu,--force-device-scale-factor=1,--window-size=568,1024
onBefore.js has run for tablet_v.
onReady.js has run for:  tablet_v
Starting Chromy: port:9224 --disable-gpu,--force-device-scale-factor=1,--window-size=1024,768
onBefore.js has run for tablet_h.
onReady.js has run for:  tablet_h
      COMMAND | Executing core for `report`
      compare | The comparison failed with error: Error: ENOENT: no such file or directory, open 'C:\Source\backstop_data\bitmaps_mytest\20170825-153105\failed_diff_test_BackstopJS_Homepage_0_document_0_phone.png'
      COMMAND | Command `report` ended with an error after [0.005s]
      COMMAND | TypeError: Cannot read property 'failed' of undefined
                    at C:\Source\multi-tenant\mdp.node.seleniumtests\node_modules\backstopjs\core\command\report.js:98:26
      COMMAND | Command `test` ended with an error after [3.94s]
      COMMAND | TypeError: Cannot read property 'failed' of undefined
                    at C:\Source\multi-tenant\mdp.node.seleniumtests\node_modules\backstopjs\core\command\report.js:98:26

npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "test"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! backstopjs@3.0.20 test: `node ./cli/index.js test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the backstopjs@3.0.20 test script 'node ./cli/index.js test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the backstopjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./cli/index.js test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs backstopjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls backstopjs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Source\multi-tenant\mdp.node.seleniumtests\node_modules\backstopjs\npm-debug.log
```
From what I can see this is pointing to the wrong directory somehow.
The path to backstop_data is `C:\Source\multi-tenant\mdp.node.seleniumTests\backstop_data\`
and the reference and test screenshots both show up in that `backstop_data`.
for reference backstop is `C:\Source\multi-tenant\mdp.node.seleniumTests\node_modules\backstopjs`

But as you can see above, the error happens when it's looking in the directory `C:\Source\backstop_data\` which does not match the successful calls above.

This is also a little surprising because this is the first test in the instructions, so there shouldn't be any failing screenshots?

Am I misunderstanding the config file or is this actually a bug?  Any help is appreciated.  I've attached the npm-debug file and the backstop.json at the root of the project.

Thank you.

[npm-debug.txt](https://github.com/garris/BackstopJS/files/1253159/npm-debug.txt)

[backstop.json.txt](https://github.com/garris/BackstopJS/files/1253160/backstop.json.txt)

**Comment by garris** [comment]
Hi @nurovi Thank you for posting!  We just released version 3.0 which has breaking changes.  The CSS-Tricks article is out of date. Please check the instructions here for set up and usage...  https://github.com/garris/BackstopJS/

Please note -- the new set up does not want you to cd into the node_modules directory.  Easiest way to use is just do a global install and cd to your project to run the config from there.  

I will be rewriting that CSS-TRICKS article soon.  Sorry for any confusion!

**Comment by nurovi** [comment]
Thank you for the speedy reply! I'll go through those instructions.
